app/gui2/src/components/GraphEditor/widgets/WidgetNumber.vue

```diff
@@ -62,14 +62,14 @@ export const widgetDefinition = defineWidget(
 </script>
 
 <template>
-  <!-- See comment in GraphNode next to dragPointer definition about stopping pointerdown and pointerup -->
   <NumericInputWidget
     ref="inputComponent"
     v-model="value"
     class="WidgetNumber r-24"
     :limits="limits"
     @pointerdown.stop
     @pointerup.stop
```

Comment:
Another one. Do we still want to stop those?

Reply:
Some are needed because the `NumericInputWidget` does not stops clicks itself (similarly to HTML's input element).